### ISSUE TITLE
Missing accessibility identifiers for lockIcon shield images

### ISSUE THREAD

**Issue by dragosb01** [mozilla-mobile/firefox-ios#32393]
The lockIcon shield from the addressBar can have 2 states, with protection and without, cases where the images are different.
On automation we need different accessibility identifiers in order to perform validation for these 2 states.
This ticket is a request for these identifiers to be added.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-15043)
